modules/rosa-sts-account-wide-roles-and-policies.adoc

```diff
@@ -0,0 +1,777 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-about-iam-resources.adoc
+
+[id="rosa-sts-account-wide-roles-and-policies_{context}"]
+= Account-wide IAM role and policy reference
+
+This section provides details about the account-wide IAM roles and policies that are required for ROSA deployments that use STS, including the Operator policies. The JSON files that define the policies are also provided.
+
+The account-wide roles and policies are specific to an OpenShift version, for example OpenShift 4.8. You can minimize the required STS resources by reusing the account-wide roles and policies for multiple clusters of the same version.
```

Comment:
Should we specify that it's the same minor/y-stream version? For example, 4.8.3 and 4.8.12 are considered the same version for the purpose of these policies.